src/content/api/introduction.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: Introduction - Draft
+description: Draft Introduction
+nav:
+  path: API/Getting Started
+  order: 1
+---
+
+Setting `draft: true` in the front matter of the guide means this page will not be published in production. This allows us to incrementally migrate the legacy API reference. We will remove this once we migrate the first api reference page.
```

Comment:
@AndyClifford Suggestion: Use the content in https://docs.centrapay.com/api/introduction